test/unit/math/prim/functor/ode_ckrk_prim_test.cpp

```diff
@@ -0,0 +1,314 @@
+#include <stan/math/prim.hpp>
+#include <gtest/gtest.h>
+#include <stan/math/prim/functor/ode_ckrk.hpp>
+#include <boost/numeric/odeint.hpp>
+#include <test/unit/math/prim/functor/harmonic_oscillator.hpp>
+#include <test/unit/math/prim/functor/lorenz.hpp>
+#include <test/unit/util.hpp>
+#include <iostream>
+#include <sstream>
+#include <vector>
+#include <limits>
+
+std::ostream* msgs = nullptr;
```

Comment:
We want to avoid using global variables in tests as we use the so-called jumbo tests in Jenkins. This means that all 
prim/functor tests are built as a single file, which speeds up testing by a factor.

If global variables cant be avoided, put them in a unique namespace like ode_ckrk_test::.

rev/functor does not use jumbo tests (https://github.com/stan-dev/math/blob/develop/runTests.py#L37) so its not an issue there.

Reply:
thx @rok-cesnovar . fixed it.